schemas/secret.yml

```diff
@@ -0,0 +1,19 @@
+id: http://schemas.taskcluster.net/secrets/v1/secret.json#
+$schema:  http://json-schema.org/draft-04/schema#
+title:        "A TaskCluster Secret"
+description: |
+  Message containing a TaskCluster Secret
+type:         object
+properties:
+  value:
+    description: |
+      The secret value to be encrypted.
+    type:         string
```

Comment:
Why not just use ´type: object` then it can be any JSON object... we just serialize it... In fact `base.Entity` will serialize, sign and encrypt it for you.